File: python/exercício_49.py
print()
print("=-"*60)
print()
print("--------------Numero primo ou não------------------")
print()
print("=-"*60)
print() 
n1=int(input("Digite um numero que deseja saber se é primo ou não: "))
tot = 0
for c in range(1, n1 + 1):
            if n1 % c == 0:
                tot += 1
if tot == 2:
    print("O numero {} é primo".format(n1))
else:
    print("O numero {} não é primo".format(n1))

File: java/exercício_49.java
import java.util.Scanner;

public class exercício_49 {
    public static void main(String[] args) {

        int n, c;
        boolean x;

        try (Scanner leitor = new Scanner(System.in)) {
            System.out.println("Digite um numero para saber se ele e primo ou nao: ");
            n = leitor.nextInt();
        }

        x = true;

        c = 2;

        while (x & c < n) {

            if (n % c == 0) {

                x = false;
            }

            c = c + 1;
        }

        if (x) {

            System.out.println("O numero digitado e PRIMO");
        } else {

            System.out.println("O numero digitado nao e PRIMO");
        }
    }

}
